packages/axelar-wasm-std/src/voting.rs

```diff
@@ -191,7 +191,7 @@ impl Poll for WeightedPoll {
         &mut self,
         block_height: u64,
         sender: &Addr,
-        votes: &[bool],
+        votes: Vec<bool>,
     ) -> Result<PollStatus, Error> {
         if !self.snapshot.is_participant(sender) {
```

Comment:
```
        let participant = self
            .snapshot
            .participants
            .get(&sender.to_string())
            .ok_or(Error::NotParticipant {})?;
```

Reply:
I generally avoid access fields directly. participants can be private and encapsulated. only make fields public for necessary public interface. 

But we can agree on our standard practices.